lib/kafo/hook_context.rb

```diff
@@ -22,14 +22,20 @@ def logger
     end
 
     # if you want to add new app_option be sure to do as soon as possible (usually boot hook)
-    # otherwise it may be to late (e.g. when displaying help)
+    # otherwise it may be too late (e.g. when displaying help)
```

Comment:
It felt like overkill to create a 2nd PR or commit to fixup this comment. Since it's a related helper anyway, I included it here.